src/Nethermind/Nethermind.Consensus/Withdrawals/WithdrawalProcessor.cs

```diff
@@ -42,7 +42,14 @@ public void ProcessWithdrawals(Block block, IBlockTracer blockTracer, IReleaseSp
             {
                 if (_logger.IsTrace) _logger.Trace($"  {withdrawal.AmountInGwei} GWei to account {withdrawal.Address}");
 
-                witness.AccessCompleteAccount(withdrawal.Address);
+                var gasAvailable = block.GasLimit - block.GasUsed;
+                var gasBefore = gasAvailable;
+                if (!witness.AccessCompleteAccount(withdrawal.Address, ref gasAvailable))
+                {
+                    throw new OutOfGasException();
```

Comment:
there was no gas charge here - we just add witness here, without charging the gas

Reply:
fixed